python-frontend/src/main/java/org/sonar/python/tree/TreeUtils.java

```diff
@@ -252,4 +254,14 @@ public static boolean isBooleanLiteral(Tree tree) {
     }
     return false;
   }
+
+  /**
+   Returns true if the call expression is not an instance call.
+   */
+  public static boolean isStaticCall(CallExpression callExpression) {
+    return callExpression.callee().is(Tree.Kind.NAME) || Optional.of(callExpression.callee())
```

Comment:
Indeed! Thanks!
For the current check, it wouldn't matter because we would not recognize `f` as a function symbol so we wouldn't risk an FP there.
However I will remove it from the utility method since it could lead to a wrong behavior in a different context.